sdk/deepbook-v3/src/utils/config.ts

```diff
@@ -38,17 +34,16 @@ export class DeepBookConfig {
 
 	constructor({
 		client,
-		signer,
 		env,
 		adminCap,
+		address,
 	}: {
 		client: SuiClient;
-		signer: Signer;
 		env: Environment;
+		address: string;
 		adminCap?: string;
 	}) {
-		this.client = client;
```

Comment:
@hayes-mysten we no longer initialize the client here?

Reply:
yes, the config doesn't need a client anymore, and none of the contract methods do either